frontend/packages/dev-console/integration-tests/features/monitoring/filter-the-alerts.feature

```diff
@@ -7,8 +7,8 @@ Feature: Filter for Alert state and Severity
         Given user is at developer perspective
         And user is at Add page 
         And user has created or selected namespace "aut-monitoring-alerts"
-        # To configure the alerts on cluster - need to execute below yaml file
-        # https://gist.githubusercontent.com/vikram-raj/27fa5c9d7e0cf223919c697d34bd2beb/raw/f85ab7a0e1f3e8a6270c124a4a97df13e5b9cb3c/ns-alert-setup.yaml
```

Comment:
We can add this YAMLs into testData. The PR with YAMLs folder is ready to merge. Once the PR is merged, you can add these YAMLs to YAMLs folder in testData.

Reply:
Updated